kolibri/plugins/learn/assets/src/routes/index.js

```diff
@@ -151,9 +151,6 @@ export default [
     name: PageNames.TOPICS_CONTENT,
     path: '/topics/c/:id',
     handler: toRoute => {
-      if (unassignedContentGuard()) {
```

Comment:
Yes - with this guard in place, learners could not access a content page at all. This is a result of the deprecation and removal of the lesson specific content page in favour of `TOPICS_CONTENT` - I don't know that this was entirely intentional, but that's where we are now, so we need to allow this.

Reply:
Thank you. If we allow all learners to access a resource page, what about related pages that still have the guard in place and it's possible to navigate there from this page, for example [this route](https://github.com/learningequality/kolibri/pull/8744/files/572602504229f2eeb3d6ee6a3f0e9015098f43b1#diff-cdb2ba5c6ec7d704c3556d3c2ea149c954b0331732dcb422d349286de8421e0aR136)? Does the guard need to be removed from every related page then?

Reply:
Wondering if it'd make sense to rather revisit the guard based on new requirements when compared to removing only one of its occurrences to prevent issues that could result from having it for some routes and not having it for others.

Reply:
I'm sorry for the invalid route link, not sure what's wrong. I was referencing one route above:
```
{
    name: PageNames.TOPICS_TOPIC,
    path: '/topics/t/:id/:subtopic?',
    handler: (toRoute, fromRoute) => {
      if (unassignedContentGuard()) {
        return unassignedContentGuard();
      }
      // If navigation is triggered by a custom navigation updating the
      // context query param, do not run the handler
      if (toRoute.params.id === fromRoute.params.id) {
        return;
      }
      showTopicsTopic(store, { id: toRoute.params.id, pageName: toRoute.name });
    },
  },
```

Reply:
I think this should still be in place - when we are viewing content in the lesson context, and the unassigned content guard is active, we shouldn't be providing links to topics or non-assigned content pages. There's an open issue for this in the context of the completion modal, but I think in all other cases we are avoiding this.